lang/stata/stata.py

```diff
@@ -0,0 +1,170 @@
+from talon import Context, actions, settings
+
+ctx = Context()
+
+ctx.matches = r"""
+code.language: stata
+"""
+
+# functions.py
+ctx.lists["user.code_parameter_name"] = {
+    # regressions
+    "v c e cluster": "vce(cluster)",
+    "v c e robust": "vce(robust)",
+}
+
+# functions_common.py
+ctx.lists["user.code_common_function"] = {
+    # base stata
+    "global": "global",
+    "local": "local",
+    "reg": "reg",
+    "regress": "reg",
+    # packages
+    "estadd": "estadd",
```

Comment:
Just want to confirm you pronounce these literally "estadd" and not "E S T add". If the latter, should change the wording to spell it out.